src/import-data/ImportData.test.ts

```diff
@@ -0,0 +1,448 @@
+        mockAjax: {
+          Workspaces: {
+            workspace: getWorkspaceApi,
+          },
+        },
+        queryParams: {
+          format: 'PFB',
+          url: importUrl,
+        },
+      });
+
+      // Act
+      await importIntoExistingWorkspace(user, defaultGoogleWorkspace.workspace.name);
+
+      // Assert
+      expect(getWorkspaceApi).toHaveBeenCalledWith(
+        defaultGoogleWorkspace.workspace.namespace,
+        defaultGoogleWorkspace.workspace.name
+      );
+
+      expect(importJob).toHaveBeenCalledWith(importUrl, 'pfb', null);
+    });
+  });
+
+  describe('BagIt imports', () => {
+    it('imports BagIt file when format is unspecified', async () => {
+      // Arrange
+      const user = userEvent.setup();
+
+      const importBagit = jest.fn().mockResolvedValue(undefined);
+      const getWorkspaceApi = jest.fn().mockReturnValue({
+        importBagit,
+      });
+
+      const importUrl = 'https://example.com/path/to/file.bagit';
+      setup({
+        mockAjax: {
+          Workspaces: {
+            workspace: getWorkspaceApi,
+          },
+        },
+        queryParams: {
+          url: importUrl,
+        },
+      });
+
+      // Act
+      await importIntoExistingWorkspace(user, defaultGoogleWorkspace.workspace.name);
+
+      // Assert
+      expect(getWorkspaceApi).toHaveBeenCalledWith(
+        defaultGoogleWorkspace.workspace.namespace,
+        defaultGoogleWorkspace.workspace.name
+      );
+
+      expect(importBagit).toHaveBeenCalledWith(importUrl);
+    });
+  });
+
+  describe('Entities JSON imports', () => {
+    it('imports Rawls entities JSON', async () => {
+      // Arrange
+      const user = userEvent.setup();
+
+      const importJSON = jest.fn().mockResolvedValue(undefined);
+      const getWorkspaceApi = jest.fn().mockReturnValue({
+        importJSON,
+      });
+
+      const importUrl = 'https://example.com/path/to/file.json';
+      setup({
+        mockAjax: {
+          Workspaces: {
+            workspace: getWorkspaceApi,
+          },
+        },
+        queryParams: {
+          format: 'entitiesJson',
+          url: importUrl,
+        },
+      });
+
+      // Act
+      await importIntoExistingWorkspace(user, defaultGoogleWorkspace.workspace.name);
+
+      // Assert
+      expect(getWorkspaceApi).toHaveBeenCalledWith(
+        defaultGoogleWorkspace.workspace.namespace,
+        defaultGoogleWorkspace.workspace.name
+      );
+
+      expect(importJSON).toHaveBeenCalledWith(importUrl);
+    });
+  });
+
+  describe('TDR imports', () => {
+    describe('TDR exports', () => {
+      const queryParams = {
+        format: 'tdrexport',
+        snapshotId: '00001111-2222-3333-aaaa-bbbbccccdddd',
+        snapshotName: 'test-snapshot',
+        tdrmanifest: 'https://example.com/path/to/manifest.json',
+        tdrSyncPermissions: 'true',
+        url: 'https://data.terra.bio',
+      };
+
+      it('imports TDR exports into Google workspaces', async () => {
```

Comment:
I think the test output here will end up reading like: `TDR imports TDR exports imports TDR exports into Google workspaces` 🤯 

Maybe drop / outdent / rephrase where possible?

Reply:
Updated some describes so that this is now `TDR > snapshot exports > imports snapshot exports into Google workspaces`.